libs/dh/admin/feature-user-management/src/lib/invite-user-modal/dh-invite-user-modal.component.html

```diff
@@ -14,4 +14,74 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<watt-modal #inviteUserModal> invite user </watt-modal>
+﻿<ng-container *transloco="let t; read: 'admin.userManagement.inviteUser'">
+  <watt-modal
+    #inviteUserModal
+    [title]="t('title')"
+    (closed)="closeModal($event)"
+  >
+    <mat-stepper [linear]="true" #stepper>
```

Comment:
I would love to see a `watt-stepper` component ;) 
Suggestion: Maybe just create a task on `bounty-hunters`, and we can discuss it?

Reply:
I was thinking the same, I just wanted an implementation or two before making a general step watt component :-)

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/issues/1418